versioned/storage/cassandra/src/main/java/org/projectnessie/versioned/storage/cassandra/CassandraPersist.java

```diff
@@ -307,23 +314,31 @@ public <T extends Obj> T[] fetchTypedObjsIfExist(
   @Override
   public boolean storeObj(@Nonnull Obj obj, boolean ignoreSoftSizeRestrictions)
       throws ObjTooLargeException {
-    return writeSingleObj(obj, false, ignoreSoftSizeRestrictions, backend::executeCas);
+    long referenced = config.currentTimeMicros();
+    if (writeSingleObj(obj, referenced, false, ignoreSoftSizeRestrictions, backend::executeCas)) {
+      return true;
+    }
+    updateSingleReferenced(obj.id(), referenced, backend::execute);
```

Comment:
It's an extremely rare situation.
It can only happen, if you generate the exact same object again - e.g. a `ContentValueObj` having the same metadata pointer + snapshot ID + ...
Or a fast forward merge.